lib/ansi.rb

```diff
@@ -0,0 +1,62 @@
+module FastlaneCI
+  ##
+  # little helper class for parsing ansi to html
```

Comment:
Did you write this class, or is it taken from somewhere?

Reply:
It's original content :)